### PR TITLE
feat(CX-2886): Update SavedWorks empty state

### PR DESCRIPTION
**Pull request by MrSltun**
### Description

This PR resolves [CX-2886]

This PR updates the saved works empty state screen

### Screenshots

| Platform | Before | After |
| -- | -- | -- |
| Android | Screenshot_1664548030 | Screenshot_1664548064 |
| iOS | Simulator Screen Shot - iPhone 13 Pro Max - 2022-09-30 at 16 25 17 | Simulator Screen Shot - iPhone 13 Pro Max - 2022-09-30 at 16 24 48 |

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- update saved works empty state screen - mrsltun

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2886]: https://artsyproduct.atlassian.net/browse/CX-2886?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Favorites/FavoriteArtworks.tsx [hunk 1]
```diff
@@ -63,79 +68,55 @@ export class SavedWorks extends Component<Props, State> {
...
-            }
-          />
-        </StickyTabPageScrollView>
-      )
-    }
+  const artworks = extractNodes(me?.followsAndSaves?.artworks)
 
+  if (artworks.length === 0) {
     return (
-      <ClassTheme>
```
Comment (MrSltun) [inline src/app/Scenes/Favorites/FavoriteArtworks.tsx:112]: When refactoring from Class to Functional component, I thought that this is only used in Class components
is it okay to remove it?

File: src/app/Scenes/Favorites/FavoriteArtworks.tsx [hunk 2]
```diff
@@ -63,79 +68,55 @@ export class SavedWorks extends Component<Props, State> {
...
+      <StickyTabPageScrollView
+        refreshControl={
+          <StickTabPageRefreshControl refreshing={refreshingFromPull} onRefresh={handleRefresh} />
+        }
+        contentContainerStyle={{ flexGrow: 1, justifyContent: "center", height: "100%" }}
+      >
+        <Flex alignItems="center">
+          <Text lineHeight={32} fontSize={20}>
+            Keep track of artworks you love
+          </Text>
```
Comment (MrSltun) [inline src/app/Scenes/Favorites/FavoriteArtworks.tsx:84]: I belive that this is `md` in Palette v3, are we updating to Palette v3 soon? so I could add a comment to update this when we do

**Comment by dariakoko** [comment]
Thank you Sultan
What do you think of using ZeroState component? I've adjusted it a little bit in [this PR](https://github.com/artsy/eigen/pull/7430), we could reuse it here as well.

**Comment by dariakoko** [comment]
Let's make the title bigger, as per [the design](https://www.figma.com/file/l3plkt7gMSg4p1Or0E8SLC/My-Collection-2022?node-id=6210%3A92194)?
image

**Comment by MrSltun** [comment]
>Thank you Sultan
>What do you think of using ZeroState component? I've adjusted it a little bit in https://github.com/artsy/eigen/pull/7430, we could reuse it here as well.

> Let's make the title bigger, as per [the design](https://www.figma.com/file/l3plkt7gMSg4p1Or0E8SLC/My-Collection-2022?node-id=6210%3A92194)?
>image

@dariakoko  I like the ZeroState component, but the new bigger title is a bit bigger than the design, since it's using Palette v3.1 (maybe I'm nitpicking here 😅)
Maybe we can use it for now and add a comment to update it to match the design once we have Palette v3.1 in Eigen?

File: src/app/Scenes/Favorites/FavoriteArtworks.tsx (hunk 2 above)
Comment (pvinis) [reply src/app/Scenes/Favorites/FavoriteArtworks.tsx:84]: the PR is here. https://github.com/artsy/eigen/pull/7392. i think I will just merge it right now.

**Review by pvinis — APPROVED** [review]

**Comment by dariakoko** [comment]
@MrSltun ah oh, I see, the design's been updated since I last saw it
Updating my PR, since the Palette is already updated

**Review by dariakoko — APPROVED** [review]
Thank you Sultan ⭐x5

**Pull request merged**